### ISSUE TITLE
Merge Tags Fail after adding Vimeo Uploader to a Gravity Forms View

### ISSUE THREAD

**Issue by johnvsc** [gdarko/wp-vimeo-videos#58]
TL;DR: [watch the video explanation](https://www.zavocki.com/vimeo-error-example.mp4 )

My client has your pro version  (Michele at TunesForGood.org) and this is just what we need to go live. ... HOWEVER,  I have bizarre edge case that is blocking our launch - and it seems to happen when we render a Gravity  Forms View (grrrrrr) after we add your plugin.

We're leaning on Gravity Views (https://gravityview.co/ ) to display the entry results. GV has this great option where we can grab a value, called a Merge Tag, and print it on the display or, in our case, pass it to a short code.

When I add the Vimeo Uploader through Gravity Forms and present it to the Gravity View ... it knocks out all the merge tags. So my short code returns null (or default a default rendering) and not the specifics from the entry instance ( grrrrrrrrr ).

FYI I stepped through the code, went to [Gravity Forms documentation](https://docs.gravityforms.com/category/user-guides/merge-tags-getting-started/) and  burned some hours before I said, "Ok, now I need to contact the dev." I also Contacted GV ... and they said the same.

Thoughts?

oh, and thanks for creating a great plugin!

**Comment by johnvsc** [comment]
maybe a solution can be found on an add on that uses Merge Tags: https://docs.gravityforms.com/gf_field_coupon/

**Comment by gdarko** [comment]
Hi @johnvsc 

I am looking at the issue already. Will let you know once i have more details.

Best Regards,
Darko

**Comment by gdarko** [comment]
Hi @johnvsc  

Can you please contact me at info@codeverve.com with the login details of the test site and i will be looking asap.

I don't have GravityView set up on my dev machine so it will be definitely faster if you contact me.

Best Regards,
Darko

**Comment by gdarko** [comment]
Hey @johnvsc 

Just to notify you that i received password reset link, but your site is protected by HTTP Basic Auth and i can't proceed. Please either remove it send me the login details in email.

Best Regards,
Darko

**Comment by johnvsc** [comment]
Hi Darko - can you provide a link to the patch for this ticket ... i cant seem to find it!

Thanks - john z

**Comment by gdarko** [comment]
Marking as resolved. Version 1.8.1 resolves this issue.
